Maui.RevenueCat.InAppBilling/Platforms/Android/Extensions/PeriodTypeExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Maui.RevenueCat.InAppBilling.Enums;
+using Com.Revenuecat.Purchases;
+
+namespace Maui.RevenueCat.InAppBilling.Platforms.Android.Extensions;
+internal static class PeriodTypeExtensions
+{
+    internal static PeriodId ToPeriodId(this PeriodType period)
+    {
+        if (period == PeriodType.Intro) return PeriodId.Intro;
+        if (period == PeriodType.Trial) return PeriodId.Trial;
+        if (period == PeriodType.Normal) return PeriodId.Normal;
+        throw new ArgumentException();
+    }
```

Comment:
- `switch` again
- please rename `period` to reflect the type used -> `periodType`

Reply:
same problem preventing the use of switch.

Naming change complete.